src/components/reviewMatches/ReviewLink.tsx

```diff
@@ -194,90 +179,117 @@ const ReviewLink = () => {
-          }
+        buttons={[
+          <Button
+            startIcon={<SearchOutlined />}
+            variant="outlined"
+            size="large"
+            onClick={() => setIsSearchModalVisible(true)}
+          >
+            Refine Search
+          </Button>
         ]}
       />
       <Divider
         sx={{
-          mb: 4
+          mb: 3
         }}
       />
-
-      <Typography variant="dgSubTitle">
-        PATIENT LINKED TO GOLDEN RECORD
-      </Typography>
-      <DataGrid
-        data={data.filter((r: AnyRecord) => {
-          if (r.type === 'Golden' || r.type === 'Current') {
-            return r
-          }
-        })}
+      <Box
         sx={{
-          mb: 4
+          mb: 3,
+          borderRadius: '4px',
+          boxShadow: '0px 0px 0px 1px #E0E0E0'
         }}
-      />
-      <Typography variant="dgSubTitle">OTHER GOLDEN RECORDS</Typography>
-      <DataGrid
-        data={mapDataToScores(
-          data.filter((r: AnyRecord) => {
-            if (r.type === 'Candidate') {
+      >
+        <Typography pl={1.5} variant="dgSubTitle">
+          PATIENT LINKED TO GOLDEN RECORD
+        </Typography>
+
+        <DataGrid
+          data={matchDetails.filter((r: AnyRecord) => {
+            if (r.type === 'Golden' || r.type === 'Current') {
               return r
             }
-          }),
-          payload?.candidates
-        )}
+          })}
+          sx={{
+            '.MuiDataGrid-columnSeparator': {
+              display: 'none'
+            },
+            '& .MuiDataGrid-columnHeaders': {
+              backgroundColor: '#274263',
+              color: '#FFF',
+              borderRadius: '0px'
+            },
+            borderRadius: '0px',
+            p: 0
+          }}
+        />
+      </Box>
+      <Typography variant="dgSubTitle">OTHER GOLDEN RECORDS</Typography>
+      <DataGrid
+        data={candidateGoldenRecords || []}
         handleOpenCreateNewGRDialog={handleOpenCreateNewGRDialog}
         handleOpenLinkedRecordDialog={handleOpenLinkedRecordDialog}
+        sx={{
+          '& .MuiDataGrid-columnHeaders': { display: 'none' },
+          '& .MuiDataGrid-virtualScroller': { marginTop: '0!important' }
+        }}
       />
-
-      <Stack direction="row" sx={{ mt: 3 }} spacing={1}>
-        <Button variant="contained" onClick={() => acceptAndClose()}>
-          Accept & Close
-        </Button>
-        <Button variant="outlined" onClick={() => leavePending()}>
-          Leave as pending
+      <Stack direction="row" sx={{ mt: 3 }} justifyContent={'space-between'}>
+        <Stack direction="row" spacing={1}>
+          <Button variant="contained" onClick={() => acceptAndClose()}>
+            Accept & Close
+          </Button>
+          <Button variant="outlined" onClick={() => leavePending()}>
+            Leave as pending
+          </Button>
+        </Stack>
+        <Button
+          variant="outlined"
+          onClick={() => handleOpenCreateNewGRDialog()}
```

Comment:
```suggestion
          onClick={() => openCreateGoldenRecordDialog()}
```
If we create then it's necessarily new :)